LDAP-Auth/LDAPAuthenticationProviderPlugin.cs

```diff
@@ -19,7 +22,7 @@ namespace Jellyfin.Plugin.LDAP_Auth
     /// </summary>
     public class LdapAuthenticationProviderPlugin : IAuthenticationProvider, IPasswordResetProvider
     {
-        private readonly ILogger<LdapAuthenticationProviderPlugin> _logger;
+        private static ILogger<LdapAuthenticationProviderPlugin> _logger;
```

Comment:
Revert this change and make `GetConnectionOptions` and `LdapClient_UserDefinedServerCertValidationDelegate` non-static

Reply:
This would also require a trickle-up of making `LdapConnectionOptions` non-static. Is that ok with you, or would you prefer I remove the extra logging from `LdapClient_UserDefinedServerCertValidationDelegate`?

Reply:
Yes, please make the function `GetConnectionOptions` non-static as originally stated

Reply:
Done. Thanks for the review!